### DISCUSSION TITLE
Winners are announced!

### DISCUSSION THREAD

**Discussion by pamelafox** [microsoft/AI-Chat-App-Hack#138] [Announcements]
We've announced the winners here:
https://techcommunity.microsoft.com/t5/ai-azure-ai-services-blog/announcing-the-winners-of-the-first-ai-chat-app-hack/ba-p/4067840
John will be in touch with the winners over email.

It was very hard to choose, as there were so many great hacks. We'll be showcasing them all in a blog post on https://techcommunity.microsoft.com/t5/educator-developer-blog/bg-p/EducatorDeveloperBlog tomorrow, so stay tuned to that. Also look forward to your Credly badge coming soon. Thanks for all your hard work!

**Comment by crissins** [comment]
Hello Pam, 

Is there an update in regards to the Credly badges?
